src/views/ContinuationOut.vue

```diff
@@ -120,7 +161,7 @@
               </header>
               <div :class="{ 'invalid-section': !certifyFormValid && showErrors }" id="certify-form-section">
                 <Certify
-                  :class="{ 'invalid-certify': !certifyFormValid && showErrors }"
+                  :class="{ 'invalid-component': !certifyFormValid && showErrors }"
                   ref="certifyRef"
                   :isCertified.sync="isCertified"
                   :certifiedBy.sync="certifiedBy"
```

Comment:
Line 169: provide an appropriate certify text that matches the note in the uxpin ("It is an offence...")